lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -202,7 +204,7 @@ def va_profile_opt_in_out_lambda_handler(event: dict, context, worker_id=None) -
         headers.get("Authorization", headers.get("authorization", '')),
         integration_testing_public_cert if is_integration_test else va_profile_public_cert
     ):
-        logger.error("Authentication failed.")
+        logger.info("Authentication failed.  Returning 401.")
```

Comment:
It will get logged either way, but I changed it to "info" because it's not an error in the sense that the code or system is misperforming.  The error is client side.